apps/nuxt3-ssr/components/harmonization/HarmonizationTableCellAvailableIcon.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
+import type { HarmonizationStatus } from "~/interfaces/types";
+
+const props = defineProps<{
+  status: HarmonizationStatus;
+}>();
+
+const availableStatuses = ["partial", "complete"];
+
+const recodedStatus = computed(() => {
+  if (availableStatuses.includes(props.status)) {
+    return "available";
+  }
+  return props.status;
```

Comment:
this feels somewhat indirect , why not pass 'available' as status instead of this computation at the Icon level

Reply:
Removed